generator/integration_tests/golden/golden_thing_admin_client.cc

```diff
@@ -21,6 +21,7 @@
 #include <memory>
 #include "generator/integration_tests/golden/golden_thing_admin_options.h"
 #include <thread>
+#include "google/cloud/internal/operation_id.h"
```

Comment:
nit: mm... I guess `clang-format` is disabled in this directory, but that sounds like an unfortunate order

Reply:
Right ... we do not format the golden files.

But if we're going to rely on a post formatting stage at all in order to make generation simpler/easier, then I would assert that this falls into that category.  If we want to complicate the generator, then we should get rid of post formatting altogether.